python/src/aiconfig/editor/client/src/components/prompt/prompt_input/attachments/AttachmentContainer.tsx

```diff
@@ -41,7 +41,7 @@ export default memo(function AttachmentContainer({
       </Flex>
       <MimeTypeRenderer
         mimeType={attachment.mime_type}
-        content={attachment.data as string}
+        content={(attachment.data as AttachmentDataWithStringValue).value as string}
```

Comment:
Nice. In a followup PR, I think it would be good to add 'kind' as a prop (can default to "file_uri") to MimeTypeRenderer and pass it in here and in PromptOutputsRenderer then handle the `data:${output.mime_type};base64,` prefix within MimeTypeRenderer; for PromptOutputsRenderer, we'd be able to get rid of a bit of code to make it
```
      case "file_uri":
      case "base64":
        return (
          <PromptOutputWrapper
            copyContent={(output.data as OutputDataWithValue).value as string}
            output={output}
            withRawJSONToggle
          >
            <MimeTypeRenderer
              mimeType={output.mime_type}
              content={(output.data as OutputDataWithValue).value as string}
              kind={output.data.kind as "file_uri" | "base64"}
            />
          </PromptOutputWrapper>
        );
```
instead of 2 separate cases